cmd/meroxa/root/apps/deploy.go

```diff
@@ -401,8 +409,6 @@ func (d *Deploy) buildApp(ctx context.Context) error {
 	switch d.lang {
 	case GoLang:
 		err = turbineGo.BuildBinary(ctx, d.logger, d.path, d.appName, true)
-	case JavaScript:
```

Comment:
There is no _true_ turbine-js "build" step anymore. Node applications are essentially shipped as is.